core/src/main/java/com/linecorp/armeria/internal/common/NonWrappingRequestContext.java

```diff
@@ -151,10 +151,6 @@ public final SessionProtocol sessionProtocol() {
         return sessionProtocol;
     }
 
-    protected void sessionProtocol(SessionProtocol sessionProtocol) {
```

Comment:
This method is not used internally. I don't think we have to open an API to alter the existing `SessionProtocol`.

Reply:
Thanks for the cleanup 👍